packages/admin/cms-admin/src/pages/pageTree/useCopyPastePages.tsx

```diff
@@ -259,14 +292,41 @@ function useCopyPastePages(): UseCopyPastePagesApi {
                     throw Error("Did not receive new uuid for page tree node");
                 }
 
-                // 1c. Create new document
+                // 1c. Copy all files used on page to target scope
+                const { data: filesOnPage } = await client.query<GQLGetAllFilesUsedOnPageQuery, GQLGetAllFilesUsedOnPageQueryVariables>({
```

Comment:
It would be nicer to only execute this section if the rootScope equals the targetScope. That way we could avoid two requests.

However, I couldn't find a way to get the scope of the page in the clipboard.